xtask/src/launcher.rs

```diff
@@ -32,6 +32,7 @@ use crate::internal::*;
 pub enum LauncherMode {
     Qemu,
     Crosvm,
+    Binary,
```

Comment:
Matter of taste, but I don't like the term 'binary' for this -- it's not like the other two aren't binaries. 'Linux' would be more descriptive, or maybe even 'native'.

Reply:
I wouldn't go with linux, since one can also run linux inside one of the VMs (see experimental/oak_functions_loader_linux_vsock). I suppose native works, can go with that

Reply:
Ah yes, we have linux overloaded already as well :(
In that case, yes, I think 'Native' would be a better name.